README.md

```diff
@@ -11,6 +12,24 @@ Run `yarn`
 
 Run `yarn start`
 
+### Launch dev dependencies
+
+To test locally you will need a few dependencies: Kafka, Clickhouse and OpenHIM. Use the following command to setup those in a dev environment.
```

Comment:
Curious to know why you set it up outside of the platform and not suggesting it's tested with platform dev mode?

Reply:
I figured this was the easiest and quickest way to launch the dependencies. It starts in a few seconds as it skips all the checks and config that the platform does. I think the docker compose way still makes sense for testing single components but for testing full solutions the platform is a more complete tool.